estate/models/estate_property.py

```diff
@@ -0,0 +1,52 @@
+from odoo import models, fields
+from datetime import datetime, timedelta
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Estate property model"
+
+    name = fields.Char(string='Name',
+                       required=True,
+                       help='This is the name of the estate property.',
+                       index=True)
```

Comment:
one line is file here, or make the indentation as you do below, but not like this